portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,16 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds comments from /data
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+async function getComments() {
+  const commentsData = await fetch("/data");
+  const comments = await commentsData.json();
```

Comment:
I don't think you need this second await

Reply:
Resolved because json() also returns another promise.
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch